emrun.py

```diff
@@ -151,7 +159,7 @@ def logi(msg):
     if emrun_options.log_html:
       sys.stdout.write(format_html(msg))
     else:
-      print(msg, file=sys.stdout)
+      print_to_handle(sys.stdout, msg)
```

Comment:
does `from __future__ import print_function` not take care of this?

Reply:
It did not, still needed this.